page-stories.hbs

```diff
@@ -53,6 +53,9 @@
             </div>
         </div>
     </div>
+    <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js" integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+    <script src="{{asset "js/bootstrap-pagination.js"}}"></script>
+    <script src="{{asset "js/filter.js"}}"></script>
```

Comment:
Putting `<script>` tags in the middle of the document is generally bad because it blocks rendering of content after the script tag. You can probably shift the script tags here `default.hbs` using something like [`{{#has}}`](https://ghost.org/docs/themes/helpers/has/).